### ISSUE TITLE
[SB Admin Pro] Dark Mode

### ISSUE THREAD

**Issue by davidtmiller** [StartBootstrap/pro-feedback#10]
I've had an email request asking if SB Admin Pro comes with a dark mode. This is something worth looking into. Please +1 this comment if you're interested in seeing a dark mode color palette developed for SB Admin Pro.

**Comment by GertJanFrl** [comment]
Would like that! And prefer to have it with media queries not a special class.

**Comment by davidtmiller** [comment]
Yes it looks like the media query dark/light preference is pretty widely supported in modern browsers: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by davidtmiller** [comment]
This has been added to the SB Admin Pro v1.3.0 project milestone.

I will look into the current best practices regarding implementing dark mode and go from there.

**Comment by GertJanFrl** [comment]
@davidtmiller Any update on this? Because 1.3 is already released. (No pressure intended)

**Comment by davidtmiller** [comment]
This was shelved due to the Bootstrap 5 release which took priority over a dark mode feature, but we will be circling back to this along with better RTL support.

**Comment by GertJanFrl** [comment]
Hi David,

Any update on this? Is there anything needed from our sides?

Thanks!

**Comment by davidtmiller** [comment]
Not at the moment, but with the Bootstrap 5.2.1 update we will look into getting this added in.

**Comment by LauLaman** [comment]
Now with bootstrap 5.3 also supporting dark mode any news on this?

**Comment by GertJanFrl** [comment]
Any update on this @davidtmiller ?

**Comment by davidtmiller** [comment]
Hi there all, we are working on updating SB Admin Pro to introduce the dark mode features that come with 5.3, so yes dark mode will be with the upcoming 5.3 update!

**Comment by StefanoPaparesta** [comment]
Any update on this @davidtmiller ?

**Comment by felixschilling** [comment]
Push @davidtmiller

**Comment by GertJanFrl** [comment]
@davidtmiller, please be real. Is there going to be an update or not? The last update seems to be from 2022-03-28. 

If not I will transition to another template

**Comment by davidtmiller** [comment]
We've yet to develop the 5.3 update. The plan is to release the 5.3.3 update for the themes in Q2 of this year. Regular updates have been on hold for some time, but we plan to get back to releases this year. Sorry for the delay on this one for everyone holding out for it.

**Comment by AnthonyDesert** [comment]
Hello, I'm really enjoying the SB Admin Pro theme, do you have any updates regarding the dark theme?
